.travis.yml

```diff
@@ -55,4 +55,30 @@ jobs:
         - git push "https://${GITHUB_TOKEN}@${GH_REMOTE_REPO}" master > /dev/null 2>&1
         - cd ..
         - make push-bundle-to-quay
+    - stage: Acceptance tests
+      if: type = pull_request
+      env:
+        - KUBECONFIG=$HOME/miniconfig
+        - MINIKUBE_WANTUPDATENOTIFICATION=false
+        - MINIKUBE_WANTREPORTERRORPROMPT=false
+        # exclude knative and example annotated tests
+        - EXTRA_BEHAVE_ARGS="--tags=~@knative --tags=~@examples"
```

Comment:
There is no need for extra variable (`EXTRA_BEHAVE_ARGS` for setting tag expression)... you can use existing like `TEST_ACCEPTANCE_TAGS="~@knative,~@examples"`

Reply:
Unfortunately, we need that - because `TEST_ACCEPTANCE_TAGS="~@knative,~@examples"` semantic implies logical OR, but individual `--tags` options are connected with logical AND. See the official help:

```
$ behave --tags-help
Scenarios inherit tags declared on the Feature level. The simplest
TAG_EXPRESSION is simply a tag::

    --tags @dev

You may even leave off the "@" - behave doesn't mind.

When a tag in a tag expression starts with a ~, this represents boolean NOT::

    --tags ~@dev

A tag expression can have several tags separated by a comma, which represents
logical OR::

    --tags @dev,@wip

The --tags option can be specified several times, and this represents logical
AND, for instance this represents the boolean expression
"(@foo or not @bar) and @zap"::

    --tags @foo,~@bar --tags @zap.

Beware that if you want to use several negative tags to exclude several tags
you have to use logical AND::

    --tags ~@fixme --tags ~@buggy.

```

Scenarios that we run on minikube should not have the following tags: `@disabled`, `@knative`, and `@examples` and only way to achieve that on Behave 1.2.6 is to use logical AND, i.e. 

`--tags=~@disabled --tags=~@knative --tags=~@examples`

In Behave 1.2.7 the tag syntax is improved, but that version is not released yet.

Reply:
Ah, I assumed that we would use the "tag expressions" described [here](https://behave.readthedocs.io/en/latest/tag_expressions.html).
In our case that would be: `TEST_ACCEPTANCE_TAGS="not (@knative and @examples)"`.

But didn't realize it wasn't released, yet.

However, having `EXTRA_BEHAVE_ARGS` available is useful for other purposes, anyway. So I agree to use your approach.